so_pysm_models/get_so_model.py

```diff
@@ -0,0 +1,102 @@
+import healpy as hp
+import numpy as np
+import pysm
+import os
+from astropy.utils import data
+from pysm.common import read_map, loadtxt
+
+data_dir = os.path.join(os.path.dirname(__file__), 'so_template')
+template = lambda x: os.path.join(data_dir, x)
```

Comment:
you do not need `template`, `get_pkg_data_filename` first looks into `so_pysm_models/data`, if something is there uses it, otherwise downloads it from the remote url, so you can use the same code also for the 512 templates

Reply:
Shouldn't they have the same relative path? meaning that, for example, high resolution dust maps should be put in `/global/project/projectdirs/cmb/www/so_pysm_models_data/data/dust`?

Reply:
actually, the nside 512 templates are already 100 MB total, that is a bit annoying in a Python package, let's have also the 512 templates remote.

Reply:
done